### PR TITLE
Add hacktoberfest page to layer5-ng

### PR DESCRIPTION
**Pull request by jcombs0929**
Signed-off-by: Jeff Combs j.combs.design@gmail.com

**Description**

This PR fixes #

**Notes for Reviewers**
Added target="_blank" to <a> tags that leave layer5-ng site.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ X ] Yes, I signed my commits.

### CODE REVIEW

**Comment by kishore007k** [comment]
I think the PR should point to the `layer5-ng` branch, not the `master` branch. @jcombs0929

**Comment by kishore007k** [comment]
If you find any difficulty please don't hesitate to contact me! or @Nikhil-Ladha

**Comment by jcombs0929** [comment]
Hi @Nikhil-Ladha i wouldn't mind a little help here. I think I ran into issues when trying to resolve the DCO fail.

**Comment by jcombs0929** [comment]
@Nikhil-Ladha I think I got it resolved. Yay

**Comment by Nikhil-Ladha** [comment]
@Tanuj22 can you please re-run the netlify deployment?

**Comment by Tanuj22** [comment]
> https://deploy-preview-1056--layer5ng.netlify.app

@Nikhil-Ladha this link is working right?

**Comment by Nikhil-Ladha** [comment]
Ohh, actually there are two deployments from netlify. Got confused 😆

**Comment by Nikhil-Ladha** [comment]
Well, what's the link to the page.
https://deploy-preview-1056--layer5ng.netlify.app/hacktoberfest/2020 this one doesn't work for me. 😕

**Comment by Tanuj22** [comment]
> Well, what's the link to the page.
> https://deploy-preview-1056--layer5ng.netlify.app/hacktoberfest/2020 this one doesn't work for me.

This is the link - https://deploy-preview-1056--layer5ng.netlify.app/programs/hacktoberfest/2020 :sweat_smile:

**Comment by kishore007k** [comment]
In the current layer5-ng site there is no link pointing towards the `Programs` page @Tanuj22 @Nikhil-Ladha 
It would be better if we have a link in the navbar or in the footer that points to the `Programs` page.

**Comment by kishore007k** [comment]
I can see that `GSOC` `GSOD` and `Community Bridge` is populated in the footer but not the `programs` page itself

**Comment by Nikhil-Ladha** [comment]
> In the current layer5-ng site there is no link pointing towards the `Programs` page @Tanuj22 @Nikhil-Ladha
> It would be better if we have a link in the navbar or in the footer that points to the `Programs` page.

Agreed!

**Comment by Nikhil-Ladha** [comment]
I think the resources and presenter column data is opposite @jcombs0929 . Please check that. Also, the hacktoberfest-layer5 logo size can be increased a bit.

**Comment by Jashpatel1** [comment]
> I think the resources and presenter column data is opposite @jcombs0929 . Please check that. Also, the hacktoberfest-layer5 logo size can be increased a bit.
> 
> And better to add some visual difference to the links on the page. Like color, size.

Yes, how about a table with borders for Tutorials section?

**Comment by Tanuj22** [comment]
Colors of links are controlled centrally. This should not be changed here. We can create a separate issue for that

**Comment by jcombs0929** [comment]
I have made edits and changes to the table. How would guys like for me to proceed with adding the programs page to navigation, or would you rather make that a separate issue? Just let me know!

**Comment by Jashpatel1** [comment]
> I have made edits and changes to the table. How would guys like for me to proceed with adding the programs page to navigation, or would you rather make that a separate issue? Just let me know!

We will look into it as another issue. As its position is to be discussed.

File: src/collections/programs/hacktoberfest-2020/index.mdx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+<ProgramsWrapper>
+
+<img src={hacktoberfest2020}/>
+<h2>Participate in Hacktoberfest 2020 with Layer5</h2>
+<div>
+    Hacktoberfest is open to everyone in the global community. Whether you’re
+      a developer, student learning to code, event host, or a company of any
+      size, you can help drive growth of open source and make positive
+      contributions to an ever-growing community. All backgrounds and skill
+      levels are encouraged to complete the challenge. Read more about
```
Comment (Jashpatel1) [inline src/collections/programs/hacktoberfest-2020/index.mdx:21]: There is no word space between "about Hacktoberfest". Please add that,

Comment (jcombs0929) [reply src/collections/programs/hacktoberfest-2020/index.mdx:21]: This has been fixed

**Comment by Tanuj22** [comment]
@jcombs0929 Can you center the image after a particular breakpoint. In fact, you can do it for all the same for all the pages in programs. This is happening currently - 
Screenshot from 2020-11-02 15-21-08

**Review by Tanuj22 — APPROVED** [review]
Great work @jcombs0929. Thanks for sticking with this :)

**Comment by jcombs0929** [comment]
Thank you for the opportunity!

**Pull request merged**